db/data.js

```diff
@@ -643,7 +636,7 @@ const produce = [
     imageURL:
       "https://res.cloudinary.com/dvpop7e9w/image/upload/v1659637881/Seasonal/009-kale_wuhokp.png",
     description:
-      "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed feugiat euismod augue, ac fringilla diam pharetra id. In porttitor convallis nisi vel finibus. Pellentesque at diam venenatis, ornare mi eget, vestibulum arcu. Vivamus consectetur, erat eu semper dapibus, dolor sem suscipit mi, ut dignissim enim nibh vel risus. Sed aliquet, sapien eu congue gravida, lorem metus bibendum mauris, quis maximus nibh erat eu erat. Cras vulputate purus magna, tincidunt porttitor massa vestibulum sed. Nam turpis massa, dictum eu accumsan in, posuere sit amet lorem. Phasellus est sapien, faucibus eget elit et, blandit consequat ex. Proin feugiat arcu a magna iaculis, nec pharetra orci porta. Proin vel orci ante. Aliquam viverra non ligula nec rhoncus. Ut et tincidunt elit. Phasellus non risus in mi vulputate venenatis vitae quis enim. Vestibulum quis dolor venenatis, lobortis diam in, faucibus turpis. Nullam quis pharetra nulla, vestibulum tincidunt ipsum. Donec non ultrices ligula. Quisque at interdum velit. Donec laoreet elit enim, a vulputate risus cursus eu. Duis et est in mi pulvinar rhoncus in ut lorem. Integer nec dolor et velit sollicitudin vehicula. Duis sapien justo, condimentum nec orci eu, dictum dapibus nisl. Praesent ultricies libero vel leo pulvinar molestie.",
+      "Kale is a dark green leafy vegetable, and is a member of the cabbage family. It comes in two varieties: regular kale, which has smooth leaves, and curly kale, which has crinkly leaves, and is the more common of the two varieties. Kale is very high in a lot of minerals and vitamins, and is a nutritious vegetable which can contribute strongly to a varied and balanced diet. Kale is usually enjoyed boiled or steamed, as the leaves can be somewhat disagreeable and bitter in flavour if eaten raw. However, kale can also be sliced and added to stir fries, soups and risottos, or wilted in a pan and used as a side dish. Cold-hardy and resilient, kale is a great option for growing at home in the UK climate. ",
```

Comment:
Extra space after the full stop at the end of the string?

Reply:
Thank you again aquatic king.